src/CosmosDB/CosmosDB/ChangeLog.md

```diff
@@ -19,6 +19,14 @@
 -->
 
 ## Upcoming Release
+* Update Redistribute Throughput API to support source list as optional.
```

Comment:
it looks like your base branch was based on the preview feature branch, and incorrectly targeting to main branch, please double check your target release